### DISCUSSION TITLE
Using suggester - showing field-name?

### DISCUSSION THREAD

**Discussion by mhofherr** [chhoumann/quickadd#492] [Q&A]
Hi all,

I am using suggester to pre-populate certain meta-data fields in my template. How can I show the field-name in the suggester pop-up for which the choices are associated?

Example:
> field1:: {{VALUE:yes, no, }}
> field2:: {{VALUE:yes, no, }}
> field3:: {{VALUE:yes, no, }}

Currently I see 3 pop-ups showing "yes/no"-options, but I do not know for which field.

Any ideas how to bring in text(e.g. the field name) into the suggester pop-up?

Thanks and best regards
Matthias

**Comment by chhoumann** [comment]
Coming back to this very late - sorry about that.

The `|label:` flag does exactly this now - it puts helper text on the suggester popup, so you can tell the three apart:

```markdown
field1:: {{VALUE:yes,no|label:field1}}
field2:: {{VALUE:yes,no|label:field2}}
field3:: {{VALUE:yes,no|label:field3}}
```

Two related options, in case they fit even better:

- _One-page input for choices_ (Beta, Settings -> QuickAdd) shows all three fields in a single form, each under its own name - no popup guessing at all.
- If several fields should share one answer, `|name:` lets a single pick be reused: `{{VALUE:yes,no|name:answer}}` once, then `{{VALUE:answer}}` elsewhere.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#value-label

Closing as resolved.
